decision_service/unit_test/http_server.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
is this replaced by the C++ http server?

Reply:
yes, this one can be removed